src/api/procedures/modifyClaims.ts

```diff
@@ -169,6 +169,47 @@ export async function prepareModifyClaims(
       modifyClaimArgs.map(([identityId, claim]) => tuple(identityId, claim))
     );
   } else {
+    if (operation === ClaimOperation.Add) {
+      const existingCddId: string[] = [];
+
+      const claimTargets = claims.filter(
+        ({ claim: { type } }) => type === ClaimType.CustomerDueDiligence
+      );
+
+      const issuedCddClaims = await context.issuedClaims({
```

Comment:
All of this should only be done only if `newCddClaims.length` is a positive number. Otherwise you're performing an unnecessary request